### PR TITLE
Feat/mobileui

### PR DESCRIPTION
**Pull request by vallq**
Changes:
- added aspect ratio for mobile interface

### CODE REVIEW

**Comment by baviera08** [comment]
Hi @vallq thanks for the changes! 

Adding screenshots for a quick look.

## Moto G4
### Vertical
image
### Horizontal
image

## Ipad
### Vertical
image

### Horizontal
image

**Comment by baviera08** [comment]
Just thinking out loud here, in the cases where the list is long, wouldn't be useful to have a fixed sticky title. (E.G: https://www.w3schools.com/howto/howto_js_sticky_header.asp)
sticky iun app

What do you think about it?

File: package-lock.json [hunk 1]
```diff
@@ -6467,9 +6467,9 @@
       "integrity": "sha1-ksnBN0w1CF912zWexWzCV8u5P6Q="
     },
     "http-proxy": {
-      "version": "1.18.0",
-      "resolved": "https://registry.npmjs.org/http-proxy/-/http-proxy-1.18.0.tgz",
-      "integrity": "sha512-84I2iJM/n1d4Hdgc6y2+qY5mDaz2PUVjlg9znE9byl+q0uC3DeByqBGReQu5tpLK0TAqTIXScRUV+dg7+bUPpQ==",
+      "version": "1.18.1",
+      "resolved": "https://registry.npmjs.org/http-proxy/-/http-proxy-1.18.1.tgz",
+      "integrity": "sha512-7mz/721AbnJwIVbnaSv1Cz3Am0ZLT/UBwkC92VlxhXv/k/BBQfM2fXElQNC27BVGr0uwUpplYPQM9LnaBMR5NQ==",
```
Comment (baviera08) [inline package-lock.json:6472]: Just out of curiosity, did you update some package or audit and fix?

**Comment by koonpeng** [comment]
> Just thinking out loud here, in the cases where the list is long, wouldn't be useful to have a fixed sticky title. (E.G: https://www.w3schools.com/howto/howto_js_sticky_header.asp)

Thats a great idea, we have to fix the headers getting mixed with the contents though. We also have to make sure the contents are given enough space for low res displays. Also have to make sure clicking the header can still close the expansion panel, I don't know if there is such functionality built-in in material. In any case, we can explore this in a separate PR.

**Pull request merged**